images/build/go-runner/go-runner-setcap/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the manager binary
+ARG GO_VERSION
```

Comment:
Would it be better to use go-runner as a base image for this? I don't like the fact that I have to duplicate so much of the Dockerfile. Also the entire change would be much smaller.

Reply:
if you do then this can't just be a "variant" as they're built concurrently.

Reply:
it could be built as an additional build step in the current one though

Reply:
Can you please explain more how the additional build step would look like? I'd prefer not duplicating the Dockerfile if possible.

Reply:
in cloudbuild.yaml, if we did build go-runner, then push go-runner, then build this image FROM that one

Reply:
I think I found an alternative way that does not involve adding another step in cloudbuild.yaml